src/neo/Network/P2P/Payloads/PluginPayload.cs

```diff
@@ -0,0 +1,110 @@
+using Neo.IO;
+using Neo.Ledger;
+using Neo.Persistence;
+using Neo.SmartContract;
+using Neo.SmartContract.Native;
+using Neo.SmartContract.Native.Designate;
+using System;
+using System.IO;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class PluginPayload : IVerifiable
+    {
+        public string Plugin;
```

Comment:
How about enum type for plugin type?

Reply:
The plugin must be decoupled from the core